engines/aio/Dockerfile

```diff
@@ -51,9 +52,15 @@ RUN ln /usr/bin/${env.project_artifactId}-${env.project_version}.jar /usr/bin/${
       dnf --enablerepo=devel install -y $LIBREOFFICE_ARM64_RPM_URL && \
       update-alternatives --set java java-17-openjdk.aarch64; \
     fi && \
-    curl -s -S $ALFRESCO_PDF_RENDERER_LIB_RPM_URL -o alfresco-pdf-renderer-linux.tgz && \
-    tar xf alfresco-pdf-renderer-linux.tgz -C /usr/bin && \
-	rm -f alfresco-pdf-renderer-linux.tgz && \
+    if [ "$(uname -m)" = "x86_64" ]; then \
+      curl -s -S $ALFRESCO_PDF_RENDERER_LIB_RPM_URL -o alfresco-pdf-renderer-linux.tgz && \
+      tar xf alfresco-pdf-renderer-linux.tgz -C /usr/bin && \
+      rm -f alfresco-pdf-renderer-linux.tgz; \
+    else \
+      curl -s -S $ALFRESCO_PDF_RENDERER_ARM64_LIB_RPM_URL -o alfresco-pdf-renderer-linux.tgz && \
+      tar xf alfresco-pdf-renderer-linux.tgz -C /usr/bin && \
+      rm -f alfresco-pdf-renderer-linux.tgz; \
+    fi && \
```

Comment:
The only thing that changes between the if and the else branch is the URL, we could avoid repetition by simply setting the location depending on the identified architecture, and then always running:

```shell
      tar xf alfresco-pdf-renderer-linux.tgz -C /usr/bin && \
      rm -f alfresco-pdf-renderer-linux.tgz; \
```
regardless of the chosen URL _(similar to what you have done in the other Dockerfile)_.